web/src/components/Socials.tsx

```diff
@@ -0,0 +1,34 @@
+import { FaFacebookF, FaInstagram, FaLinkedin } from "react-icons/fa";
+import { IoMdMail } from "react-icons/io";
+
+export default function Socials() {
+	return (
+		<>
```

Comment:
Try avoid using data-testid. Better to replace this with something screen readers can pick up at a very minimum